hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,3 @@
+RELEASE_TYPE: patch
+
+This patch refactors some internals, continuing our work on supporting alternative backends (:issue:`3086`). There is no user-visible change.
```

Comment:
Ah, we're failing tests because this is exactly identical to an existing changelog entry!  Let's push a newline...